src/support/lockedpool.cpp

```diff
@@ -224,6 +224,13 @@ PosixLockedPageAllocator::PosixLockedPageAllocator()
     page_size = sysconf(_SC_PAGESIZE);
 #endif
 }
+
+// Some systems (at least OS X) do not define MAP_ANONYMOUS yet and define
+// MAP_ANON which is deprecated
```

Comment:
If MAP_ANON is deprecated, what is the new name?

Also: this passed Travis, how is that possible if it didn't exist on MacOSX?